website/docs/guide-chapter-instructions.rst

```diff
@@ -460,31 +461,31 @@ normalized to lie between zero and one.
    W_n = \left[ {\frac{\sum_{i=1}^{I} {\sum_{j=1}^{J}{D_{ij}^2 / p_i } q_j } }{\min (I - 1,J - 1)}} \right]^{\frac{1}{2}} = \left[ {\frac{X_{LD}^2 / 2N}{\min (I - 1,J - 1)}}\right]^{\frac{1}{2}}
 
 
-When there are only two alleles per locus, :math:`W_n` is equivalent
-to the correlation coefficient between the two loci, defined as
-:math:`r =\sqrt {D_{11} / p_1 p_2 q_1 q_2 }`.
+   When there are only two alleles per locus, :math:`W_n` is equivalent
```

Comment:
```suggestion
two alleles case
   When there are only two alleles per locus, :math:`W_n` is equivalent
```

sphinx apparently expects a new paragraph after the end of the `:math:` block otherwise it thinks this paragraph is part of the math, so making a new heading here